llvm/test/CodeGen/AMDGPU/hsa-metadata-queue-ptr-v4.ll

```diff
@@ -0,0 +1,101 @@
+; RUN: opt -mtriple=amdgcn-amd-amdhsa -mcpu=gfx700 -passes=amdgpu-attributor -o %t.gfx7.bc %s
```

Comment:
Can we autogen the check lines please?

Reply:
tldr no. This is the minimal effort of the port of the v5 test, didn't bother tracking down where we lost the coverage.

The generated checks only cover the instructions in the function, and not the metadata in the function prolog (or at the summary printed at the end of the module)